### PR TITLE
Added A New Score Card

### PR DESCRIPTION
**Pull request by mujafar7**

### CODE REVIEW

File: src/pages/CocosGame.tsx [hunk 1]
```diff
@@ -168,36 +172,43 @@ const CocosGame: React.FC = () => {
...
+              score={gameResult.detail.gameScore}
+              message={"You Completed the Lesson"}
               showDialogBox={showDialogBox}
-              yesText={"Yes"}
-              noText={"No"}
-              handleClose={(e:any) => {
-                setShowDialogBox(false);
-                saveTempData(gameResult.detail, undefined);
-                push();
+              yesText={"Like the Game" }
```
Comment (bhanugummana) [inline src/pages/CocosGame.tsx:187]: Add the text to translation json

**Review by bhanugummana — CHANGES_REQUESTED** [review]

File: src/components/parent/ScoreCard.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import { chevronForward } from "ionicons/icons";
+import { title } from "process";
+import { IoCloseCircle } from "react-icons/io5";
+import { PAGES } from "../../common/constants";
+import ScoreCard from "../LessonCardStarIcons";
+import StarImages from "./ScoreCardStarIcons";
+import { AiTwotoneHeart } from "react-icons/ai";
+import { GrFormNext } from "react-icons/gr";
+
+const NewScoreCard: React.FC<{
```
Comment (bhanugummana) [inline src/components/parent/ScoreCard.tsx:22]: where are you using this NewScoreCard.
if you are using it somewhere change the name to ScoreCard

File: src/components/parent/ScoreCard.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            
+            <div className="score-card-content">
+            <div className="score-card-content-message" >{message}</div>
+            <div className="score-card-content-lesson-name" >{lessonName}</div>
+            </div>
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions
+          style={{
+            justifyContent: "space-around",
```
Comment (bhanugummana) [inline src/components/parent/ScoreCard.tsx:92]: put this in css

File: src/components/parent/ScoreCard.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            // padding: "8% 5%",
+            textAlign: "center",
+            width: width,
+            height: height,
+          }}
+        >
+          <DialogContentText
+            style={{
+              textAlign: "center",
+              color: "black",
```
Comment (bhanugummana) [inline src/components/parent/ScoreCard.tsx:71]: put this style in css file

**Review by bhanugummana — CHANGES_REQUESTED** [review]

File: src/components/parent/ScoreCardStarIcons.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import "./ScoreCardStarIcons.css";
+
+const StarImages: React.FC<{
```
Comment (Sutara-Skanda) [inline src/components/parent/ScoreCardStarIcons.tsx:3]: change the component name also. component and file name should match

**Review by Sutara-Skanda — CHANGES_REQUESTED** [review]
change the component name also. component and file name should match

**Review by Sutara-Skanda — APPROVED** [review]

**Review by bhanugummana — CHANGES_REQUESTED** [review]

**Review by bhanugummana — APPROVED** [review]

**Review by Sutara-Skanda — APPROVED** [review]

**Pull request merged**